lib/ts/recipe/dashboard/constants.ts

```diff
@@ -37,6 +37,7 @@ export const USERROLES_REMOVE_PERMISSIONS_API = "/api/userroles/role/permissions
 export const CREATE_EMAIL_PASSWORD_USER = "/api/user/emailpassword";
 export const CREATE_PASSWORDLESS_USER = "/api/user/passwordless";
 
+export const LIST_TENANT_LOGIN_METHODS = "/api/tenants/login-methods";
```

Comment:
why is this for all tenants and not just the current tenant? It should be just for the currently selected tenant.